src/core/unit.c

```diff
@@ -6111,6 +6116,33 @@ int unit_get_dependency_array(const Unit *u, UnitDependencyAtom atom, Unit ***re
         return (int) n;
 }
 
+int unit_get_dependency_chain(Unit *u, UnitDependencyAtom atom, Set **ret) {
```

Comment:
is "chain" the right word? it's a whole subgraph after all, not just one path in it as the name "chain" would suggest.

maybe unit_get_transitive_dependency_set() or so?

i mean, that's what matters here, that we transitively pick up all deps.

Reply:
'transitively'? I am not familiar with the word. I prefer 'chain' here. Or, how about unit_get_dependency_subgraph() or unit_get_graph_dependency_set() ? That is, 'chain' -> 'graph' or 'subgraph'?

Reply:
dependency_set() sounds ok too
subgraph() too

anything better than chain I think, since a chain suggests 1-dimenstional i think.

Reply:
Ah, transitive relation! I did not know the name of the relation in English. Yeah, it sounds good.